references/detection/README.md

```diff
@@ -57,7 +57,30 @@ labels.json
      ...
 }
 ```
+If you want to train a model with multiple classes, you can use the following format
```

Comment:
This should be a bit more clear:
multiclass box a ['printed', 'handwritten'] or box a ['printed'] box b ['handwritten']

Reply:
ok i think now i got it :sweat_smile: 

for example class_name_1 == 'printed' contains all boxes for printed and class_name_2 == 'handwritten' contains all boxes for handwritten.

Maybe a better format:

polygons: [....]
classes: ['printed', 'handwritten', 'printed', 'printed', ...] 

So we can avoid to introduce a new dataset format then it should also be possible to avoid the breaking change only to add a new key value pair like 'class': 'printed' @aminemindee @odulcy-mindee wdyt ?